README.md

```diff
@@ -227,6 +227,22 @@ custom_lint:
       some_parameter: "some value"
 ```
 
+#### Overriding lint error severities
+
+You can also override the severity of lint rules in the `analysis_options.yaml` file.
+This allows you to change INFO level lints to WARNING or ERROR, or vice versa:
+
+```yaml
+custom_lint:
+  errors:
+    my_lint_rule: error # Override severity to ERROR
+    another_rule: warning # Override severity to WARNING 
+    third_rule: info # Override severity to INFO
+    fourth_rule: none # Suppress the lint entirely
```

Comment:
It is redundant to list every option, and add a comment every time. The list
is already available after the code snippet, and the comment adds low value. (it reads like `final name; // The name`)

```suggestion
    my_lint_rule: error
```